health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/UploadDataMapping.js

```
@@ -348,6 +390,222 @@ function UploadDataMapping({ formData, onSelect, currentCategories }) {
+
+    // Translate properties keys and their 'name' fields
+    Object.keys(schema?.properties).forEach((key) => {
+      const translatedKey = t(key);
+      const translatedProperty = { ...schema.properties[key], name: t(schema.properties[key].name) };
+      newProp[translatedKey] = translatedProperty;
+    });
+    const newRequired = schema?.required.map((e) => t(e));
+
+    newSchema.properties = newProp;
+    newSchema.required = newRequired;
+    delete newSchema.unique;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)